java/gradle/ice.gradle

```diff
@@ -55,7 +55,17 @@ slice {
 }
 
 spotless {
+  format 'misc', {
+    // define the files to apply `misc` to
+    target '*.java', '*.gradle'
+
+    // define the steps to apply to those files
+    trimTrailingWhitespace()
+    endWithNewline()
+  }
   java {
+    toggleOffOn()
```

Comment:
This lets us add `spotless:on` and `spotless:off` in the future to skip certain blocks of code.
I didn't add it anywhere right now, but now that formatting is part of the build, we may need a way to turn it off in the future.